test/dialect/mysql/test_reflection.py

```diff
@@ -778,89 +778,140 @@ def test_nullable_reflection(self, metadata, connection):
         ).first()
         explicit_defaults_for_timestamp = row[1].lower() in ("on", "1", "true")
 
-        reflected = []
-        for idx, cols in enumerate(
-            [
-                [
+        if connection.dialect._is_mariadb_102:
+            current_timestamp = "current_timestamp()"
+        else:
+            current_timestamp = "CURRENT_TIMESTAMP"
+
+        test_cases = [
```

Comment:
**Michael Bayer** (zzzeek) wrote:

so if we were going to refactor this test, we should do it using `@testing.combinations`.

Here's the options for how we can proceed on this part.


1. we merge it as is, then I will go in after the fact and reorganize this into multiple, independent tests using `@testing.combinations`
2. the submitter Georg reorganizes this using `@testing.combinations` directly.  This is some new API to learn so I dont want to impose this on you unless you were super interested.
3.  I take over the gerrit here (we're not very good at updating github PRs) and I finish this up inline.   since I can work extremely quickly

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5102

Reply:
I'm not particularly keen on reorganizing the test case, so not having to do (2) would be nice. Otherwise, I don't really mind whether it's merged before or after

Reply:
no worries

Reply:
**Michael Bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5102